tee-worker/litentry/core/data-providers/src/graphql.rs

```diff
@@ -83,6 +83,35 @@ impl From<EvmNetwork> for VerifiedCredentialsNetwork {
 	}
 }
 
+impl From<AssertionNetwork> for VerifiedCredentialsNetwork {
```

Comment:
If I'm allowed to be a bit picky: VC = Verifiable Credential.

So it should be either `VerifiableCredentialNetwork` or `VCNetwork` or maybe `SupportedNetwork`, and please add a comment in the struct definition why we need this (or what's the difference from `AssertionNetwork`).

What's even better is that we have a trait for each data provider, something like `get_supported_network`, which returns a subset of `AssertionNetwork` => but we can do it later